src/content/plugins/eval-source-map-dev-tool-plugin.md

```diff
@@ -35,11 +35,26 @@ The following options are supported:
 
 T> Setting `module` and/or `columns` to `false` will yield less accurate source maps but will also improve compilation performance significantly.
 
+T> When using this plugin in [development mode](/concepts/mode/#mode-development), you must manually set `devtool: false`.
```

Comment:
This is misleading, it should say that when you want to use this plugin with custom configuration, it is not possible to do so without setting `devtool: false` in development mode. Using this plugin in development mode through shortcut should work just fine e.g. `devtool: 'eval-source-map'`